brokerapi/brokers/account_managers/service_account_manager.go

```diff
@@ -43,10 +44,15 @@ type ServiceAccountManager struct {
 }
 
 // creates a new service account for the given binding id with the role listed in details.Parameters["role"]
-func (sam *ServiceAccountManager) CreateCredentials(instanceID string, bindingID string, details models.BindDetails, instance models.ServiceInstanceDetails) (models.ServiceBindingCredentials, error) {
+func (sam *ServiceAccountManager) CreateCredentials(instanceID string, bindingID string, details brokerapi.BindDetails, instance models.ServiceInstanceDetails) (models.ServiceBindingCredentials, error) {
 	client := sam.HttpConfig.Client(context.Background())
 
-	role, ok := details.Parameters["role"].(string)
+	bindParameters := map[string]interface{}{}
+	if err := json.Unmarshal(details.RawParameters, &bindParameters); err != nil {
+		return models.ServiceBindingCredentials{}, err
+	}
```

Comment:
This JSON stuff will be replaced later with a proper programming-language style resolution table where variables will be resolved in the order `instance details -> bind params -> operator provided defaults -> hard-coded defaults`